openc3-cosmos-init/plugins/packages/openc3-tool-common/src/services/openc3-api.js

```diff
@@ -335,7 +335,7 @@ export class OpenC3Api {
       }
```

Comment:
Line 333 has a bug: `Invalid data type ${packet_name}` ... it should be `${value_type}`.

Reply:
This isn't a bug.  packet_name is the second positional argument, and this case is for the two argument case.